start.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Starting the containers..."
+
+source bin/cross_platform_utils.sh
+
+setupCrossPlatformEnvironment
+
+docker compose -f $(getDockerCompose) --env-file ../.env up -d --wait
```

Comment:
For it to work properly you must change the `.env` location to `.` instead of `../` .